crates/red_knot_python_semantic/src/types/signatures.rs

```diff
@@ -237,6 +237,13 @@ impl<'db> Signature<'db> {
         }
     }
 
+    pub(crate) fn bottom() -> Self {
```

Comment:
I'm curious how you came up with the name? I understand that `Never` is the bottom type, but `(…) -> Never` is a gradual type which does not participate in subtyping. Did you call it "bottom" because every arrow type is assignable to it?

Reply:
Good catch, I actually meant for this to be a `(*args: object, **kwargs: object) -> Never` signature, which is the fully-static "bottom" Callable type (subtype of all other callable types), but wrongly used the gradual form instead. Will fix.